addons/event_sale/wizard/event_configurator.py

```diff
@@ -11,7 +11,9 @@ class EventConfigurator(models.TransientModel):
 
     product_id = fields.Many2one('product.product', string="Product", readonly=True)
     event_id = fields.Many2one('event.event', string="Event")
-    event_ticket_id = fields.Many2one('event.event.ticket', string="Event Ticket")
+    event_ticket_id = fields.Many2one('event.event.ticket', string="Ticket Type",
+        compute="_compute_event_ticket_id", readonly=False, store=True)
+    available_event_ticket_count = fields.Integer("# Available Events", compute="_compute_available_event_ticket_count")
```

Comment:
Is it the number of available events or the number of available tickets? Field name is somehow not aligned with label ^^